schema/common/defaults/configuration/image-registry.yml

```diff
@@ -9,242 +9,254 @@ specification:
   images_to_load:
     x86_64:
       generic:
-        - name: "epiphanyplatform/keycloak:14.0.0"
-          file_name: keycloak-14.0.0.tar
-        - name: "rabbitmq:3.8.9"
-          file_name: rabbitmq-3.8.9.tar
-        - name: "kubernetesui/dashboard:v2.3.1"
-          file_name: dashboard-v2.3.1.tar
-        - name: "kubernetesui/metrics-scraper:v1.0.7"
-          file_name: metrics-scraper-v1.0.7.tar
-        # postgres
-        - name: bitnami/pgpool:4.2.4
-          file_name: pgpool-4.2.4.tar
-        - name: bitnami/pgbouncer:1.16.0
-          file_name: pgbouncer-1.16.0.tar
-        # ceph
-        - name: "rook/ceph:v1.8.8"
-          file_name: ceph-v1.8.8.tar
+        applications:
```

Comment:
After this change I think we need to adjust Ansible code (`image_registry` and `repository` roles) where `specification.images_to_load` is used.